src/skai/beam_utils.py

```diff
@@ -84,17 +82,20 @@ def _get_setup_file_path():
   return str(pathlib.Path(__file__).parent.parent / 'setup.py')
 
 
+
```

Comment:
nit: Can you remove these empty lines at 85 and 87? Our internal linter will complain :P

Reply:
Hi Jihyeon, I already have a version of the code that addresses all these lint issues. Just haven't pushed to github yet.

Reply:
Hi, just to confirm, no extra line at all? I keep thinking, we leave an extra line after the last line of code

Reply:
@panford There should be a newline at the end of each file. Our blank lines convention is 2 blank lines between top-level function definitions and 1 blank line between class method definitions.

See https://android.googlesource.com/platform/external/google-styleguide/+/refs/tags/android-s-beta-2/pyguide.md#3_5-blank-lines

In general the style guide above is mostly consistent with our internal style guide. It would be really helpful if you can follow it. Thanks!